src/views/landingpage/landingPageMobile.js

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import InjectSheet from 'react-jss';
+import ReactNotification from 'react-notifications-component';
+import MobileNavigationBar from '../../components/navigationbar/mobilenavigationbar';
+import { bitcoinNetwork, litecoinNetwork } from '../../constants';
+import { generateKeys, navigation } from '../../actions';
+import { MobileSwapTab } from '../../components/swaptab';
+import View from '../../components/view';
+import BackGround from '../../components/background';
+import LandingPageWrapper from './landingpagewrapper';
+
+// TODO: toggleModal,
+// TODO: isOpen,
```

Comment:
Either open issues for these `TODO`s or please describe more clearly what you mean